applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -20,68 +20,81 @@
 # Simple patch tests
```

Comment:
could u do a very small patch test example for your shells in the format of test_patch_test_small_strain (that is, all in one file...)